File: app/src/main/java/bassamalim/hidaya/database/TelawatDB.java
package bassamalim.hidaya.database;

import androidx.room.Embedded;

public class TelawatDB {

    /*@Embedded
    TelawatRecitersDB reciters;

    @Embedded
    TelawatVersionsDB versions;*/

    private int reciter_id;
    private String reciter_name;
    private String rewaya;
    private String url;
    private int count;
    private String suras;

    public TelawatDB() {}

    public TelawatDB(TelawatDB in) {
        reciter_id = in.reciter_id;
        reciter_name = in.reciter_name;
        rewaya = in.rewaya;
        url = in.url;
        count = in.count;
        suras = in.suras;
    }

    public void setReciter_id(int reciter_id) {
        this.reciter_id = reciter_id;
    }

    public void setReciter_name(String reciter_name) {
        this.reciter_name = reciter_name;
    }

    public void setRewaya(String rewaya) {
        this.rewaya = rewaya;
    }

    public void setUrl(String url) {
        this.url = url;
    }

    public void setCount(int count) {
        this.count = count;
    }

    public void setSuras(String suras) {
        this.suras = suras;
    }

    public int getReciter_id() {
        return reciter_id;
    }

    public String getReciter_name() {
        return reciter_name;
    }

    public String getRewaya() {
        return rewaya;
    }

    public String getUrl() {
        return url;
    }

    public int getCount() {
        return count;
    }

    public String getSuras() {
        return suras;
    }
}

File: app/src/main/java/bassamalim/hidaya/database/dbs/TelawatDB.kt
package bassamalim.hidaya.database.dbs

class TelawatDB(reciter_id: Int, version_id: Int, reciter_name: String, rewaya: String,
                url: String, count: Int, suras: String) {
    private var reciterId = 0
    private var versionId = 0
    private var reciterName: String
    private var rewaya: String
    private var url: String
    private var count = 0
    private var suras: String

    init {
        this.reciterId = reciter_id
        this.versionId = version_id
        this.reciterName = reciter_name
        this.rewaya = rewaya
        this.url = url
        this.count = count
        this.suras = suras
    }

    fun setReciterId(reciter_id: Int) {
        this.reciterId = reciter_id
    }

    fun setVersionId(version_id: Int) {
        this.versionId = version_id
    }

    fun setReciterName(reciter_name: String) {
        this.reciterName = reciter_name
    }

    fun setRewaya(rewaya: String) {
        this.rewaya = rewaya
    }

    fun setUrl(url: String) {
        this.url = url
    }

    fun setCount(count: Int) {
        this.count = count
    }

    fun setSuras(suras: String) {
        this.suras = suras
    }

    fun getReciterId(): Int {
        return reciterId
    }

    fun getVersionId(): Int {
        return versionId
    }

    fun getReciterName(): String {
        return reciterName
    }

    fun getRewaya(): String {
        return rewaya
    }

    fun getUrl(): String {
        return url
    }

    fun getCount(): Int {
        return count
    }

    fun getSuras(): String {
        return suras
    }
}